modules/dice_replica.R

```diff
@@ -0,0 +1,27 @@
+# DICE Damage function on level of economic productivity
+# Uses DICE-2016R
+
+#http://www.econ.yale.edu/~nordhaus/homepage/homepage/DICE2016R-091916ap.gms
+# economic impact, damage coefficient = D/(1+D) #DICE 2016R
+# T is the temperature of atmosphere increase (degrees C from 1900)
+# Damfrac, D = 0.00236Tˆ2 Damage as fraction of gross output
+# damage function = 1 - damfrac
+# Damages = Ygross * damfrac in trillions of USD per year
+# Ynet = Ygross * (damage function) output net in trillions of USD per year
+
+# Dataset accessed 2021-07-22 at https://data.giss.nasa.gov/gistemp/
+# contains temperature anomalies in Celcius compared to mean from 1951-1980
+
+temp_csv = fread(file.path('data','NASA_temp_change_1900-2020.csv'))
+temp_anomaly = select(temp_csv, "Year","J-D") # select yearly anomaly 
```

Comment:
This line doesn't work and isn't needed. You are only extracting the precise two values from the table anyway